src/Components/App.jsx

```diff
@@ -12,25 +13,27 @@ class App extends React.Component {
 
   componentDidMount() {
     const setState = (value) => {
-      this.setState({data: value});
+      this.setState({data: value });
     }
     console.log('Component Mounted');

```

Comment:
This is a useful console.log for testing during implementation, but consider removing it once this is working.